my_2d_robot/robo2d.launch

```diff
@@ -2,7 +2,13 @@
         <param name="robot_description" command="cat $(find my_2d_robot)/urdf/my2drobo.urdf" />
         <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
         <node name="my_2d_robo_state_publisher" pkg="my_2d_robot" type="my_2d_robot_node" />
+        <node name="joint_states_subscriber" pkg="my_2d_robot" type="my_2d_robot_subscriber_node" />
+        <param name="Theta1_offset" type="double" value ="10.0" />
+        <param name="Theta2_offset" type="double" value ="5.0" />  
+        <param name="link_1" type="double" value ="5.0" /> 
+        <param name="link_2" type="double" value ="5.0" /> 
         <param name="loop_rate" type="double" value="2.0" />
```

Comment:
For some reason, I could not speed up the simulation on my end. LMK if you can obtain higher than 2Hz on your laptop.

Reply:
Hmmm, I can't obtain anything higher or lower.
Will look into fixing this.

Reply:
Could be fixed in a separate PR, immediately after this one.